salt/_auth/kubernetes_rbac.py

```diff
@@ -58,6 +58,30 @@ def _check_k8s_creds(kubeconfig, token):
         raise
 
 
+def _get_groups(kubeconfig):
```

Comment:
Works for now, since we only support one group, but I would have imagined something that checks whether to add the 'node-admins' group and only return this information, and be able to call a bunch of such single-focus checks to compute the list/set of groups for a token.

Reply:
Ok make sense i will update it

Reply:
Done